lms/views/onedrive.py

```diff
@@ -0,0 +1,15 @@
+from pyramid.view import view_config
+
+
+@view_config(
+    request_method="GET",
+    route_name="onedrive.filepicker.redirect_uri",
+    renderer="lms:templates/onedrive.html.jinja2",
+)
+def redirect_uri(_request):
+    """
+    Return a basic HTML page with One Drive's JS sdk loaded.
```

Comment:
```suggestion
    Return a basic HTML page with One Drive's JS SDK loaded.
```

Reply:
OneDrive should be a single word (in contrast to Google Drive).